tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -106,5 +113,41 @@ async def test_cancel(tmp_path: Path, realization):
     sch.kill_all_jobs()
     await scheduler_task
 
-    assert (tmp_path / "a").exists()
-    assert not (tmp_path / "b").exists()
+    assert (tmp_path / str(realization.iens) / "a").exists()
+    assert not (tmp_path / str(realization.iens) / "b").exists()
+
+
+def test_add_dispatch_information_to_jobs_file(storage, tmp_path: Path):
+    test_ee_uri = "ws://test_ee_uri.com/121/"
+    test_ens_id = "test_ens_id121"
+    test_ee_token = "test_ee_token_t0k€n121"
+    test_ee_cert = "test_ee_cert121.pem"
+
+    ensemble_size = 10
+
+    ensemble = storage.create_experiment().create_ensemble(
+        name="foo", ensemble_size=ensemble_size
+    )
+    realizations = [
+        create_stub_realization(ensemble, tmp_path, iens)
+        for iens in range(ensemble_size)
+    ]
+
+    sch = scheduler.Scheduler()
+    sch.set_ee_info(test_ee_uri, test_ens_id, test_ee_cert, test_ee_token)
+
+    for realization in realizations:
+        sch.add_realization(realization, callback_timeout=lambda _: None)
```

Comment:
I think the callback_timeout defaults to None so you don't have to specify it.

Reply:
I tried, and it does not default to None. It is not in use yet, so should I make it default to None or pass in the lambda None?

Reply:
ah, there is a missing `= None` in the scheduler.py.